RELEASE-NOTES.md

```diff
@@ -58,6 +58,8 @@
 * Fixed `snow git execute` with `/` in name of the branch.
 * `snow app` commands don't enforce ownership of the objects they manage, and rely on RBAC instead.
 * `snow app deploy` for package entity now allows operating on application packages created outside the CLI
+* Add more readable error messages in `snow object create` command.
```

Comment:
Do we need this PR in 3.0?

Reply:
As we decided during discussion - this will be a part of 3.0.1/3.1.0